docs/source/release_notes.rst

```diff
@@ -17,6 +17,8 @@ v0.5.0 Jul 7, 2021
     * Enhancements
         * Add support for numpy array inputs to Woodwork (:pr:`1023`)
         * Add support for pandas.api.extensions.ExtensionArray inputs to Woodwork (:pr:`1026`)
+        * Add support for automatically inferring the ``EmailAddress`` logical
+          type (:pr:`1047`)
```

Comment:
All the updates should be placed in the `Future Release` section above, which will need to be uncommented since this is the first PR for the next release. Also, I think we should be fine to have this as the only release notes entry - if you want to mention the config param, you can do so here as well, but we've tended to keep these entries fairly brief.

Reply:
Sorry, this resulted from me improperly resolving a merge conflict.  I had originally placed them in the future releases section.